package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "legal-docs",
+  "description": "This repository contains all legal documents and their applicable translations. The repository is organize within the `docs/` directory, where each subdirectory represents a single legal document. Within a document directory is a series of markdown files, each named for the language the document is written in. For instance, `docs/privacy_policy/de.md` is the German version of the Marketplace privacy policy.",
```

Comment:
That was auto-added from the README.md. So I fixed the typo in README.md and neutered the package.json `description` to something a lot shorter since we don't need a short story in package.json.